src/routes/Send.tsx

```diff
@@ -825,6 +830,11 @@ export function Send() {
                             <p>{i18n.t("send.hodl_invoice_warning")}</p>
                         </InfoBox>
                     </Show>
+                    <Show when={source() === "onchain"}>
+                        <InfoBox accent="blue">
+                            {i18n.t("send.warn_address_reuse")}
+                        </InfoBox>
+                    </Show>
```

Comment:
this needs to be on the receive side, not send side

Reply:
okay thank you, fixed, moved to receive QR screen